java/common/src/main/scala/org/typelevel/otel4s/java/TextMapPropagatorImpl.scala

```diff
@@ -39,4 +39,18 @@ private[java] class TextMapPropagatorImpl[F[_]: Sync](
     Sync[F].delay(
       jPropagator.inject(toJContext(ctx), carrier, fromTextMapSetter)
     )
+
+  def injected[A](ctx: Vault, carrier: A)(implicit
+      injector: TextMapUpdater[A]
+  ): A = {
+    var injectedCarrier = carrier
+    jPropagator.inject[Null](
+      toJContext(ctx),
+      null, // explicitly allowed per opentelemetry-java, so our setter can be a lambda!
```

Comment:
exciting!

Reply:
I am a little terrified of a propagator calling toString in a log or short circuiting, but it's annotated nullable and explained as such in the scaladoc... and it works on the ones I've tried!

Reply:
I love how the type parameter to `jPropagator.inject` becomes `Null` because of this. it looks so silly